docs/source/releases/v1.1.rst

```diff
@@ -73,6 +109,17 @@ Misc
 Migrations
 ~~~~~~~~~~
 
+.. warning::
+   This release contains a data migration for category slugs. It will get
+   run automatically if you haven't forked the catalogue app. If you have
+   forked it, it is critical you run a copy of that migration when upgrading.
+
+* Catalogue:
+  - This app contains model changes. They should be picked up automatically.
+  - This app contains a data migration that you must run:
+    ``0003_data_migration_slugs``. It shouldn't matter whether it is executed
+    before or after the schema migration.
```

Comment:
What do you mean by 'It shouldn't matter whether it is executed before or after the schema migration.", and which schema migrations does that sentence refer to?